capstone/src/main/webapp/script.js

```diff
@@ -15,9 +15,13 @@
 function onSignIn(googleUser) {
   var profile = googleUser.getBasicProfile();
   var id_token = googleUser.getAuthResponse().id_token;
-  console.log(profile.getEmail() + " has signed in!");
-
-  window.location.href = "form.html";
+  return fetch("/login", {method: "POST", body: id_token}).then((resp) => {
+        if (resp.ok){
```

Comment:
We're going to fix this in our bad data PR. Having this "if (resp.ok)"  can cause errors because there's a chance that "resp" is undefined.

Reply:
`if (resp?.ok)` Would be a bit safer and not too much of a change for this PR.

Reply:
Got it